aat/src/aat/resources/features/F-081/F-081.feature

```diff
@@ -0,0 +1,44 @@
+@F-081
+Feature: CCD Definition Store Api : Get Jurisdiction Details
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-347 #RDM-6858
+  Scenario: must return 200 with List of jurisdictions
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains a valid jurisdiction id]
+    And it is submitted to call the [Get Jurisdiction Details] operation of [CCD Definition Store]
+    And the response [has the 200 OK code]
+    And the response [contains the jurisdictions details for jurisdiction id BEFTA_CASETYPE_2_1]
```

Comment:
This wording is a specific reference to a jurisdiction. The functional spec by a BA, on the other hand, would be in my general terms.

Reply:
And the response [contains the jurisdictions details for jurisdiction id provided to the request]